.gitignore

```diff
@@ -3,7 +3,7 @@
 #
 # Binaries for programs and plugins
 *.so
-authd
+/authd
```

Comment:
I don’t think this change is necessary personnally (I know Gabriel asked for it), but I have the habit to build the binary in the subdirectory too and would prefer for it to be ignored (we can git add any folder with that name explicitely if needed)

Reply:
fine by me as long as the 2 are ignored :)